templates/certified/index.html

```diff
@@ -6,250 +6,632 @@
 
 {% block outer_content %}
 
-<section class="p-strip--suru-bottomed">
+<section class="p-strip--suru-shape-dark is-x-deep">
   <div class="row u-vertically-center">
-    <div class="col-7">
-      <h1>Ubuntu Certified Hardware</h1>
-      <p class="p-heading--4">Machines you can trust with Ubuntu</p>
-      <p>Ubuntu certified hardware has passed our extensive testing and review process, ensuring that Ubuntu runs well out of the box, ready for your organisation. We work closely with OEMs to make Ubuntu available on a wide range of desktops, laptops, servers, IoT and SoC hardware.</p>
-      <p><a href="/certified/why-certified">Why get certified?</a></p>
-    </div>
-    <div class="col-5 u-hide--medium u-hide--small u-align--center">
+    <div class="col-3 col-small-1 col-medium-1">
```

Comment:
Maybe it was decided not to be hidden in small screens? I was just wondering as we usually hide images on small screens 🙂

Reply:
Yep, we wanted to show the logo on all screens